02 Requiements/UseCases/UseCase1SendRequest(Fully).md

```diff
@@ -0,0 +1,17 @@
+Special Requirements | --
+Technology & Data Variations List | --
+Frequency of Occurrence | The use case occurs everytime a Consultant creates a request.
+Miscellaneous | -- 
```

Comment:
Anders said that in the misc. of the fully dressed UC, you would input the feature extension such as customary orders/requests, I don't know how he wants to have it presented in the misc. tab, but maybe in the same order as the extensions, or you could link/reference to sub-UC's or something alike.